src/containers/AccountConnection.jsx

```diff
@@ -141,11 +141,30 @@ class AccountConnection extends Component {
       messages.push({message: 'account config details', params: {folder: account.folderPath}})
     }
 
+    if ([
+      'orange',
+      'orangelivebox',
+      'orangemobile'
+    ].indexOf(account.type) !== -1) {
+      messages.push(
+        {message: 'account config delay', params: {delayInDays: 15}}
+      )
+    }
     this.handleSuccess(account, messages)
   }
 
   handleUpdateSuccess (account) {
-    this.handleSuccess(account, [{message: 'account update success'}])
+    const messages = [{message: 'account update success'}]
+    if ([
+      'orange',
+      'orangelivebox',
+      'orangemobile'
+    ].indexOf(account.type) !== -1) {
+      messages.push(
```

Comment:
* Why care about the Notifier ? It will soon be removed, at least for the account management part (see MC-32).
  * Are we force to hardcode connector slugs ? It is so error prone for the future developement, to maintain every occurrence of the slug. Couldn't we instead specified a specific success message for those konnectors ? In the `manifest` or at list in `initKonnector.json` ? The condition should just be :
```js
if (this.props.connector.successMesssage && this.props.connector.successMessageDelay) {
  messages.push({message: this.props.connector.successMessage, delay: this.propse.connector.successMessageDelay})
}
```